src/main/java/com/conveyal/analysis/controllers/BundleController.java

```diff
@@ -327,13 +327,17 @@ private Bundle getBundle (Request req, Response res) {
      * and is considered the definitive source of truth. The entry in the database is a newer addition and has only
      * been around since September 2024.
      */
-    private TransportNetworkConfig getBundleConfig (Request request, Response res) {
+    private TransportNetworkConfig getBundleConfig(Request request, Response res) {
```

Comment:
Different style settings in IDE?

Reply:
I guess so? I did not intentionally change that. Similarly with the moving import, probably from "organize imports" on save.